client/src/components/DateTime/TimeSelect.js

```diff
@@ -36,12 +36,21 @@ const TimeSelect = ({ selectionSegments, onPress }) => {
           />
         </View>
       ))}
+      <View style={styles.timeSelectChild}>
+        <ButtonBox
+          text="{{ New Custom }}"
+          subtext="Enter New Time"
+          onPress={() => onPressNewSegment()}
```

Comment:
Can this be passed without the fat arrow function?

Reply:
Yes - good pickup :)